tests/app/organisation/test_rest.py

```diff
@@ -505,14 +505,54 @@ def test_post_link_service_to_organisation(admin_request, sample_service):
         organisation_id=organisation.id,
         _expected_status=204
     )
-
     assert len(organisation.services) == 1
     assert sample_service.organisation_type == 'central'
+
+
+def test_post_link_service_to_organisation_inserts_annual_billing(admin_request, sample_service):
+    data = {
+        'service_id': str(sample_service.id)
+    }
+    organisation = create_organisation(organisation_type='central')
+    assert len(organisation.services) == 0
+    assert len(AnnualBilling.query.all()) == 0
+    admin_request.post(
+        'organisation.link_service_to_organisation',
+        _data=data,
+        organisation_id=organisation.id,
+        _expected_status=204
+    )
+
     annual_billing = AnnualBilling.query.all()
     assert len(annual_billing) == 1
     assert annual_billing[0].free_sms_fragment_limit == 150000
 
 
+def test_post_link_service_to_organisation_rollback_service_if_annual_billing_update_fails(
```

Comment:
Could you help me understand the difference between this test and the one starting on line 625? They look the same to me. 

Perhaps this one was meant for `services/test_rest.py`. The `test_create_service_should_create_service_if_annual_billing_query_fails` test there needs updating to (a) not expect the service to be created (assuming that's what we want); and (b) use `side_effect`, instead of `raises` - it's still passing when it shouldn't.